google/cloud/bigquery/job/load.py

```diff
@@ -170,7 +170,7 @@ def destination_encryption_configuration(self, value):
 
     @property
     def destination_table_description(self):
-        """Optional[str]: Name given to destination table.
+        """Optional[str]: Description given to destination table.
```

Comment:
I'm not a native speaker, but this doesn't sound right? We don't really "give" description to an object.

```suggestion
        """Optional[str]: Description of the destination table.
```

Reply:
you're right, fixed this with last commit